pom.xml

```diff
@@ -401,11 +405,15 @@
 
         <!-- Maxmind GeoIp API -->
         <dependency>
```

Comment:
PS: Maybe add a comment for this dependency that it's our adaptation

Reply:
Fixed

Reply:
Awesome!